resources/views/layouts/app.blade.php

```diff
@@ -41,9 +43,9 @@
                         <!-- Authentication Links -->
                         @guest
                             <li class="nav-item">
-                                <a class="nav-link" href="{{ route('login') }}">{{ __('Login') }}</a>
+                                <a class="nav-link" href="{{ route('login') }}">{{ __('Logout') }}</a>
                             </li>
-                            @if (Route::has('register'))
+                            @if (Route::has(    'register'))
```

Comment:
please remove the space before 'register'

Reply:
I removed this space.